src/Acuminator/Acuminator.Analyzers/DiagnosticsShortName.resx

```diff
@@ -0,0 +1,324 @@
+              </xsd:sequence>
+              <xsd:attribute name="name" use="required" type="xsd:string" />
+              <xsd:attribute name="type" type="xsd:string" />
+              <xsd:attribute name="mimetype" type="xsd:string" />
+              <xsd:attribute ref="xml:space" />
+            </xsd:complexType>
+          </xsd:element>
+          <xsd:element name="assembly">
+            <xsd:complexType>
+              <xsd:attribute name="alias" type="xsd:string" />
+              <xsd:attribute name="name" type="xsd:string" />
+            </xsd:complexType>
+          </xsd:element>
+          <xsd:element name="data">
+            <xsd:complexType>
+              <xsd:sequence>
+                <xsd:element name="value" type="xsd:string" minOccurs="0" msdata:Ordinal="1" />
+                <xsd:element name="comment" type="xsd:string" minOccurs="0" msdata:Ordinal="2" />
+              </xsd:sequence>
+              <xsd:attribute name="name" type="xsd:string" use="required" msdata:Ordinal="1" />
+              <xsd:attribute name="type" type="xsd:string" msdata:Ordinal="3" />
+              <xsd:attribute name="mimetype" type="xsd:string" msdata:Ordinal="4" />
+              <xsd:attribute ref="xml:space" />
+            </xsd:complexType>
+          </xsd:element>
+          <xsd:element name="resheader">
+            <xsd:complexType>
+              <xsd:sequence>
+                <xsd:element name="value" type="xsd:string" minOccurs="0" msdata:Ordinal="1" />
+              </xsd:sequence>
+              <xsd:attribute name="name" type="xsd:string" use="required" />
+            </xsd:complexType>
+          </xsd:element>
+        </xsd:choice>
+      </xsd:complexType>
+    </xsd:element>
+  </xsd:schema>
+  <resheader name="resmimetype">
+    <value>text/microsoft-resx</value>
+  </resheader>
+  <resheader name="version">
+    <value>2.0</value>
+  </resheader>
+  <resheader name="reader">
+    <value>System.Resources.ResXResourceReader, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <resheader name="writer">
+    <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <data name="PX1000" xml:space="preserve">
+    <value>InvalidPXActionHandlerSignature</value>
+  </data>
+  <data name="PX1001" xml:space="preserve">
+    <value>PXGraphCreateInstance</value>
+  </data>
+  <data name="PX1002" xml:space="preserve">
+    <value>MissingTypeListAttributeAnalyzer</value>
+  </data>
+  <data name="PX1003" xml:space="preserve">
+    <value>NonSpecificPXGraphCreateInstance</value>
+  </data>
+  <data name="PX1004" xml:space="preserve">
+    <value>ViewDeclarationOrder</value>
+  </data>
+  <data name="PX1005" xml:space="preserve">
+    <value>TypoInViewDelegateName</value>
+  </data>
+  <data name="PX1006" xml:space="preserve">
+    <value>ViewDeclarationOrder</value>
```

Comment:
Should it differ from PX1004? If yes, we can use
for PX1004, ViewDeclarationOrderTwoCacheInstances
for PX1006, ViewDeclarationOrderOneCacheInstance